lib/environment_helpers/version.rb

```diff
@@ -1,3 +1,3 @@
 module EnvironmentHelpers
-  VERSION = "1.0.1"
+  VERSION = "1.0.2"
```

Comment:
I've been bumping minor versions for new features, but you can just leave it and I'll release 1.1.0 after both merge.